test/index.coffee

```diff
@@ -44,7 +45,36 @@ describe 'Adding files', ->
   it 'should add relationship file to link between Word and HTML files', ->
     expect(@data.word._rels['document.xml.rels']).to.be.defined
     expect(String @data.word._rels['document.xml.rels']).to
-      .match /Target="\/word\/afchunk.htm" Id="htmlChunk"/
+      .match /Target="\/word\/afchunk.mht" Id="htmlChunk"/
+
+describe 'Coverting HTML to MHT', ->
+  it 'should convert HTML source to an MHT document', ->
+    htmlSource = '<!DOCTYPE HTML><head></head><body></body>'
+    expect(utils.getMHTdocument(htmlSource)).to.match
+    /^MIME-Version: 1.0\nContent-Type: multipart\/related;/
+
+  it 'should fail if HTML source is not a string', ->
+    htmlSource = {}
+    expect(utils._prepareImageParts.bind(null, htmlSource)).to.throw /Not a valid source provided!/
+
+  it 'should detect any embedded image and change its source to ContentPart name', ->
+    htmlSource = '<p><img src="data:image/jpeg;base64,PHN2ZyB..."></p>'
+    expect(utils.getMHTdocument(htmlSource)).to.match /<img src=3D"file:\/\/fake\/image0.jpeg">/
+
+  it 'should produce ContentPart for each embedded image', ->
+    htmlSource = '<p><img src="data:image/jpeg;base64,PHN2ZyB...">
+    <img src="data:image/png;base64,PHN2ZyB...">
+    <img src="data:image/gif;base64,PHN2ZyB..."></p>'
+    imageParts = utils._prepareImageParts(htmlSource).imageContentParts
+    expect(imageParts.length).to.equal 3
+    expect(imageParts[0]).to.match /Content-Type: image\/jpeg/
+    expect(imageParts[1]).to.match /Content-Transfer-Encoding: base64/
+    expect(imageParts[2]).to.have.string "Content-Location: file://fake/image2.gif"
+
```

Comment:
From what I understand, you have 3 assertions here that would pass for any of the images, but you apply it to one image at the time to avoid duplication. I think it would be more clear to have all 3 assertions for each image (for 9 assertions in total).